scripts/synapse_port_db

```diff
@@ -445,31 +473,86 @@ class Porter(object):
 
         db_conn.commit()
 
+        return db_conn
+
+    def get_sqlite_store(self):
```

Comment:
Also: probably wants to be renamed to reflect that it is building a thing rather than fetching an existing thing. suggest `build_db_store` or similar.